io.liri.BaseApp.yaml

```diff
@@ -0,0 +1,52 @@
+id: io.liri.BaseApp
+branch: stable
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: 5.11
+separate-locales: false
+modules:
+- name: qbs
+  cleanup-platform:
```

Comment:
This property only makes sense for building runtimes.

Reply:
Removed, thanks.